bitfinex/rest/restv2.py

```diff
@@ -0,0 +1,597 @@
+
+
+
+
+    # REST PUBLIC ENDPOINTS
+    def platform_status(self):
+        """
+        Get the current status of the platform. Maintenance periods last for just few minutes and
+        might be necessary from time to time during upgrades of core components of our
+        infrastructure. Even if rare it is important to have a way to notify users. For a real-time
+        notification we suggest to use websockets and listen to events 20060/20061
+        https://bitfinex.readme.io/v2/reference#rest-public-platform-status
+        Response details : 	int	1=operative, 0=maintenance
+        """
+        path = "v2/platform/status"
+        response = self._get(path)
+        return response
+
+
+
+    def tickers(self, symbol_list):
+        """
+        The ticker is a high level overview of the state of the market. It shows you the current
+        best bid and ask, as well as the last trade price.It also includes information such as daily
+        volume and how much the price has moved over the last day.
+        https://bitfinex.readme.io/v2/reference#rest-public-tickers
+        """
+        assert isinstance(symbol_list, list), "symbol_list must be of type list"
+        assert symbol_list, "symbol_list must have at least one symbol"
+        path = "v2/tickers?symbols="+symbol_list[0]
+        for index in range(1, len(symbol_list)):
+            path += ","+str(symbol_list[index])
+        response = self._get(path)
+        return response
+
+    def ticker(self, symbol):
+        """
+        The ticker is a high level overview of the state of the market.It shows you the current best
+        bid and ask, as well as the last trade price.It also includes information such as daily
+        volume and how much the price has moved over the last day.
+        https://bitfinex.readme.io/v2/reference#rest-public-ticker
+        """
+        path = "v2/ticker/"+str(symbol)
+        response = self._get(path)
+        return response
+
+
+
+    def trades(self, symbol):
+        """
+        Trades endpoint includes all the pertinent details of the trade, such as price,
+        size and time.
+        https://bitfinex.readme.io/v2/reference#rest-public-trades
+        """
+        path = "v2/trades/"+symbol+"/hist"
+        response = self._get(path)
+        return response
+
+
+    def books(self, symbol, precision="P0"):
+        """
+        The Order Books channel allow you to keep track of the state of the Bitfinex order book.
+        It is provided on a price aggregated basis, with customizable precision.
+        https://bitfinex.readme.io/v2/reference#rest-public-books
+        Params :
+            symbol :The symbol you want information about. You can find the list of valid symbols
+                    by calling the /symbols endpoint.
+            precision : Level of price aggregation (P0, P1, P2, P3, R0)
+        """
+        path = "v2/book/"+symbol+"/"+precision
```

Comment:
Here too, in python 3 `format` is prefered to joining strings with a `+` sign.
```python
path = "v2/book/{symbol}/{precision}".format(
    symbol=symbol,
    precision=precision
)
```

or

```python
path = "v2/book/{}/{}".format(symbol, precision)
```